libs/sdk-ui-charts/src/charts/headline/internal/headlines/LegacyHeadline.tsx

```diff
@@ -48,19 +50,33 @@ export default class LegacyHeadline extends React.Component<IHeadlineVisualizati
         disableDrillUnderline: false,
     };
 
+    private secondaryItemTitleWrapperRef = createRef<HTMLDivElement>();
+    private afterRenderGuardTimeoutId;
+    private afterRenderCalled = false;
```

Comment:
what it purpose of this guard? never set to true afterRenderCalled

Reply:
fixed, and used the guard